just/_helpers.just

```diff
@@ -10,6 +10,17 @@ _check-cargo:
         exit 1
     fi
 
+[private]
+_check-docker:
+    #!/usr/bin/env bash
+
+    if ! command -v docker &> /dev/null
+    then
+        echo "docker could not be found"
+        echo "Install docker from https://docs.docker.com/get-docker/"
+        exit 1
+    fi
+
```

Comment:
Same as [here](https://github.com/Talus-Network/nexus-sdk/pull/261/files#r2222517853).

Reply:
Code is now able to handle any runtime, currently also supporting the checks for `podman` but it's extensible, see https://github.com/Talus-Network/nexus-sdk/pull/261/commits/e0667abc4ede26bf98e57dad5764bb711c54b6a1

Unfortunately, see https://github.com/Talus-Network/nexus-sdk/pull/261/commits/e0667abc4ede26bf98e57dad5764bb711c54b6a1

So I guess there needs to be follow up work, for now Docker is good enough I think.